src/components/ListItemLarge.tsx

```diff
@@ -24,7 +24,7 @@ export const ListItemLargeStyles = {
 
 const styles = StyleSheet.create(ListItemLargeStyles);
 
-const withStyle = (props: ListItemProps) => {
+const withStyle = (props: ListItemLargeProps) => {
```

Comment:
2/5 = 4/10 = 40%

This rule needs to be used in ListItemComparator

<image src="https://user-images.githubusercontent.com/24992535/95296861-c854d580-0879-11eb-9b13-86e82e78a69d.png" height="100" />

Reply:
I can not use this rule, but I changed the ComparatorSingleModal wrapper style to have a specific maxWidth (for ex. 20% in the below screen)

https://github.com/urbi-mobility/react-native-urbi-ui/blob/9c823249f0b92972509edc48a8ef1688de5f9be5/src/molecules/content/ComparatorSingleModal.tsx#L14-L15

<image src="https://user-images.githubusercontent.com/24992535/95303208-38b42480-0883-11eb-946d-ef57ff2e2c56.png" height="300" />

as displayed in the above screenshot, children elements of the view do not respect the width of the container and I'm trying to find solution for this.

https://github.com/urbi-mobility/react-native-urbi-ui/blob/9c823249f0b92972509edc48a8ef1688de5f9be5/src/molecules/Chip.tsx#L11

Reply:
Additionally behaviour of the test in the third line `No arrival estimate` does not respect rule 

<image src="https://user-images.githubusercontent.com/24992535/95304197-89784d00-0884-11eb-929f-c99673336204.png" height="50" />

https://github.com/urbi-mobility/urbi-react-native/pull/233#pullrequestreview-503578658

Reply:
commit 0c10664

<img src="https://user-images.githubusercontent.com/24992535/95314046-1c1ee900-0891-11eb-8fc0-e57925825c66.png" width="150" />

Reply:
Fixed with https://github.com/urbi-mobility/urbi-react-native/pull/233/commits/20c09081989563aa79c47cbb8e455fbf394a4a3e https://github.com/urbi-mobility/urbi-react-native/commit/20c09081989563aa79c47cbb8e455fbf394a4a3e

<img src="https://user-images.githubusercontent.com/24992535/95359600-a59edd00-08ca-11eb-8acd-95eb1c83ff62.png" width="200" />

Reply:
`This is some body text` does not respect the 2/5 rule.
Should take in considerations and expect rules explained in https://github.com/urbi-mobility/urbi-react-native/pull/233#pullrequestreview-503578658

<image src="https://user-images.githubusercontent.com/24992535/95430395-03273e00-094c-11eb-9f1d-9e13c5909963.png" width="200" />

1) Text can expand, collapse when the molecules grow
2) Text can be cutted, which means I could fix this solution with previous solution overflow hidden. https://github.com/urbi-mobility/react-native-urbi-ui/pull/17/commits/0c10664e06969f7a74e71fa1978d692dd3d1883e

<image src="https://user-images.githubusercontent.com/24992535/95430809-96607380-094c-11eb-8f81-b71de7202a48.png" height=60 />

Reply:
Solved with https://github.com/urbi-mobility/react-native-urbi-ui/pull/17/commits/8b2a339afbf9931decb8269cd4418a77e3d37adf

Reply:
https://github.com/urbi-mobility/urbi-react-native/pull/233#discussion_r501155867